### DISCUSSION TITLE
Status line  display editor indentation

### DISCUSSION THREAD

**Discussion by wiserfz** [nvim-lualine/lualine.nvim#938] [Q&A] [answered]
Is there any way to set up `lualine` display editor indentation like `vscode` or not?

Open file which indent with spaces,
<img width="285" alt="image" src="https://user-images.githubusercontent.com/38970394/211748483-5d279d37-254a-440a-9ee4-36ca5fed24cd.png">

or open file which indent with tab.
<img width="283" alt="image" src="https://user-images.githubusercontent.com/38970394/211748752-688b2855-9a12-4830-972f-e8409a65bf89.png">

Thank you for replying

**Comment by fitrh** [comment, marked as answer]
I think you can try something like this

```lua
local indent = {
    function()
        local style = vim.bo.expandtab and "Spaces" or "Tab Size"
        local size = vim.bo.expandtab and vim.bo.tabstop or vim.bo.shiftwidth
        return style .. ": " .. size
    end,
    cond = function()
        return vim.bo.filetype ~= ""
    end,
}

-- in your lualine section
-- ...
sections = {
    -- ...
    lualine_x = { indent, 'encoding', 'fileformat', 'filetype' },
    -- ...
},
-- ...
```

**Reply by wiserfz** [reply]
Thank you for answer me. I'm new for lua, this just what i want.
